packages/kbn-coloring/src/shared_components/color_mapping/color/color_handling.ts

```diff
@@ -6,25 +6,32 @@
  * Side Public License, v 1.
  */
 import chroma from 'chroma-js';
+import { findLast } from 'lodash';
```

Comment:
nit: there's a direct import for this:

```suggestion
import findLast from 'lodash/findLast';
```

Reply:
done in https://github.com/elastic/kibana/pull/175144/commits/58c37f3b9b218068f7549ef0614046d602c0ea8d